app/components/primer/beta/auto_complete.rb

```diff
@@ -102,61 +127,54 @@ class AutoComplete < Primer::Component
       #
       # @example With custom classes for the results
       #   <%= render(Primer::Beta::AutoComplete.new(label_text: "Fruits", src: "/auto_complete", input_id: "fruits-input--custom-results", list_id: "fruits-popup--custom-results")) do |c| %>
-      #     <% c.results(classes: "custom-class") do %>
-      #       <%= render(Primer::Beta::AutoComplete::Item.new(selected: true, value: "apple")) do |c| %>
-      #         Apple
-      #       <% end %>
-      #       <%= render(Primer::Beta::AutoComplete::Item.new(value: "orange")) do |c| %>
-      #         Orange
-      #       <% end %>
-      #     <% end %>
+      #     <% c.results(classes: "custom-class") %>
       #   <% end %>
       #
       # @param label_text [String] The label of the input.
       # @param src [String] The route to query.
       # @param input_id [String] Id of the input element.
       # @param input_name [String] Optional name of the input element, defaults to `input_id` when not set.
       # @param list_id [String] Id of the list element.
-      # @param with_icon [Boolean] Controls if a search icon is visible, defaults to `false`.
-      # @param is_label_visible [Boolean] Controls if the label is visible. If `false`, screen reader only text will be added.
-      # @param is_clearable [Boolean] Adds optional clear button.
-      # @param is_label_inline [Boolean] Controls if the label is inline. On smaller screens, label will always become stacked.
+      # @param visually_hide_label [Boolean] Controls if the label is visible. If `true`, screen reader only text will be added.
+      # @param show_clear_button [Boolean] Adds optional clear button.
       # @param system_arguments [Hash] <%= link_to_system_arguments_docs %>
-      def initialize(label_text:, src:, list_id:, input_id:, input_name: nil, is_label_visible: true, is_label_inline: false, with_icon: false, is_clearable: false, **system_arguments)
+      # @param size [Hash] Input size can be small, medium (default), or large
+      # @param full_width [Boolean] Input can be full-width or fit to content
+      # @param disabled [Boolean] Disabled input
+      # @param invalid [Boolean] Invalid input
+      # @param placeholder [String] The placeholder text displayed within the input
+      def initialize(label_text:, src:, list_id:, input_id:, input_name: nil, placeholder: nil, show_clear_button: false, visually_hide_label: false, size: DEFAULT_SIZE, full_width: false, disabled: false, invalid: false, **system_arguments)
```

Comment:
I generally like passing in the positive versions of flags like this, eg. `enabled: false` instead of `disabled: true` and `valid: false` instead of `invalid: true`. There's something about the inverted logic that messes with my brain. Not a blocker tho.

Reply:
We need to align on this as a team across frameworks 👍